docs/src/main/asciidoc/build-analytics.adoc

```diff
@@ -0,0 +1,156 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Build Analytics
+:categories: analytics
+:summary: This guide presents what build analytics is and how to configure it.
+
+The Quarkus team has limited knowledge, from Maven download numbers, on the excellent growth of Quarkus and the amount of users reporting issues/concerns, but we are really flying blind on which platforms, OS, java combinations, build tools etc.
```

Comment:
```suggestion
The Quarkus team has limited knowledge, from Maven download numbers, on the excellent growth of Quarkus and the amount of users reporting issues/concerns, but we are really flying blind on which platforms, OS, java combinations, build tools etc.
```
```suggestion
The Quarkus team has limited knowledge, from Maven download numbers, of the remarkable growth of Quarkus and the number of users reporting issues/concerns. Still, we need more insight into the platforms, operating system, java combinations, and build tools our users employ.
```
This suggestion:
* Uses shorter sentences.
* Avoids unfortunate terminology.
* Applies the other Vale style suggestions.